frontend/pages/SoftwarePage/components/icons/SoftwareIcon/SoftwareIcon.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { ComponentType, SVGProps } from "react";
+import {
+  SOFTWARE_NAME_TO_ICON_MAP,
+  SOFTWARE_SOURCE_TO_ICON_MAP,
+  SOFTWARE_ICON_SIZES,
+  SoftwareIconSizes,
+} from "../";
```

Comment:
We might want to move these into this file rather than index, I'll defer to Gabe.

Reply:
yes i'd agree with sarahs comment here